cvat/apps/annotation/README.md

```diff
@@ -0,0 +1,12 @@
+## Description
```

Comment:
@azhavoro , Need to add description how to write own scripts as you did for auto annotation module. Otherwise it will be difficult to understand and use.